modules/canvaskit/README_BLUESCAPE.md

```diff
@@ -0,0 +1,28 @@
+# Find a stable version to sync with
+
+Look into the npm registry https://www.npmjs.com/package/canvaskit-wasm. Start with the
+latest version. Find the git tag of this version and check the tag out. Open
+`./npm_build/package.json` to compare the version string against the version string in
+the file. If they are the same then it should be the version we want to sync with. If
+not pick the first lower version from the registry and repeat the above process until
+finding a stable version.
+
+# Sync with the stable version
+
+Create a new branch with name `bluescape_canvaskit/<upstream_version_number>` on the
+version found. Apply our patches on top of the version. Usually we cherry-pick every
```

Comment:
How does one find "our patches" to apply and ensure some weren't lost?

Reply:
We have the last synced SHA. Above patches above that SHA are ours.